python/lib/dcoscli/dcoscli/service/main.py

```diff
@@ -110,9 +110,13 @@ def _shutdown(service_id):
     :rtype: int
     """
 
-    mesos.DCOSClient().shutdown_framework(service_id)
-    return 0
-
+    services = mesos.get_master().frameworks(inactive=False, completed=False)
+    for service in services:
+        if service.dict()["id"] == service_id:
```

Comment:
Are we confident that this key is always present in the dict? Otherwise it would raise an exception, unless it is retrieved with `service.dict().get('id')`